packages/flutter/lib/src/services/image_resolution.dart

```diff
@@ -142,9 +142,9 @@ class AssetImage extends AssetBundleImageProvider {
     if (json == null)
       return null;
     // TODO(ianh): JSON decoding really shouldn't be on the main thread.
-    final Map<dynamic, dynamic> parsedManifest = JSON.decode(json);
+    final Map<String, List<String>> parsedManifest = JSON.decode(json);
     // TODO(ianh): convert that data structure to the right types.
-    return new SynchronousFuture<Map<dynamic, dynamic>>(parsedManifest); // ignore: return_of_invalid_type, https://github.com/flutter/flutter/issues/5771
+    return new SynchronousFuture<Map<String, List<String>>>(parsedManifest);
```

Comment:
This should fail in strong mode, since it's an invalid cast, as far as I can tell.